code/datums/materials/basemats.dm

```diff
@@ -161,8 +161,7 @@ Unless you know what you're doing, only use the first three numbers. They're in
 
 /datum/material/bananium/on_applied(atom/source, amount, material_flags)
 	. = ..()
-	if(!HAS_TRAIT(source, SQUEAKY_COMPONENT_TRAIT))
-		source.AddComponent(/datum/component/squeak, list('sound/items/bikehorn.ogg'=1), 50, falloff_exponent = 20)
+	source.LoadComponent(/datum/component/squeak, list('sound/items/bikehorn.ogg'=1), 50, 0, 0, 0, 0, 20, 0)
```

Comment:
I know why you stopped using the named parameters, just a shame. We should fix this someday.

Reply:
😔